src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200904133558_EES-1344-PreviousVersionIdShouldAllowNull.cs

```diff
@@ -0,0 +1,30 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.EntityFrameworkCore.Migrations;
+using static GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations.MigrationConstants;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    public partial class EES1344PreviousVersionIdShouldAllowNull : Migration
+    {
+        private const string MigrationId = "20200904133558";
+        private const string PreviousVersionMigrationId = "20200713151302";
+        
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("DROP PROCEDURE dbo.DropAndCreateRelease");
+            migrationBuilder.Sql("DROP TYPE dbo.ReleaseType");
+
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{MigrationId}_Type_ReleaseType.sql");
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{PreviousVersionMigrationId}_Routine_DropAndCreateRelease.sql");
```

Comment:
With PreviousVersionMigrationId as it stands, this wouldn't be the current version of the DropAndCreateRelease  procedure.